unit/testing-utils/require_class_not_stub.cpp

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************\
+
+ Module: Unit test utilities
+
+ Author: DiffBlue Limited. All rights reserved.
+
+\*******************************************************************/
+
+/// \file
+/// Helper functions for verifying that classes are not stubs.
+
+#include "require_class_not_stub.h"
+
+#include <testing-utils/catch.hpp>
+
+const class_typet &require_class_not_stub(
+  const symbolt &class_symbol)
+{
```

Comment:
Is a more descriptive name for this function perhaps something like `require_complete_class` ? It's not immediately obvious to me what "not_stub" means.